ibm_catalog.json

```diff
@@ -38,6 +38,7 @@
             "label": "Existing Cluster",
             "name": "ExistingCluster",
             "install_type": "extension",
+            "working_directory": "solutions/existing-cluster",
```

Comment:
Note: pointing to the DA
Your different flavors will go into the solutions folder

Reply:
ok..thanks